### DISCUSSION TITLE
Vits Output Bith Dpeth 64 Bit

### DISCUSSION THREAD

**Discussion by Rico2000** [coqui-ai/TTS#1855] [General]
Hey,

the Output of my Vits Modell has a 64 Bit Depth. How can i change this I could not find this in the docs.
I use the Synthesizer class of Coqui. 
The question is whether the Bith Depth of the model is on 64 bit or only converted afterwards. 

Thanks

**Comment by erogol** [comment]
Bit depth is something decided on saving to a file. Check ```AudioProcessor.save_wav```
